src/Rule.js

```diff
@@ -5,20 +5,24 @@ const Use = require('./Use');
 
 const Rule = Orderable(
   class extends ChainedMap {
-    constructor(parent, name) {
+    constructor(parent, name, ruleType = 'rule') {
```

Comment:
Could you rename `ruleType` to `type` (and `ruleTypes` to `types`) for parity with the naming style here:
https://github.com/neutrinojs/webpack-chain/blob/df193ecaa67f3aa0cb41c9bb4a54a63fbae7c55a/src/Plugin.js#L6-L9

Reply:
As mentioned in the comment on the PR, no. The `Rule` class already has a method called `type` (declared through the `extend` call in the constructor). 

https://github.com/neutrinojs/webpack-chain/blob/df193ecaa67f3aa0cb41c9bb4a54a63fbae7c55a/src/Rule.js#L23-L32

In 62327eb, which I pushed before running the tests, you can even see that exact change making the build fail.

Suggestions for a better name are welcome.

Reply:
How about "kind"? Only idea I've had so far.

Reply:
Ah sorry I missed that comment in the GitHub email notifications - good point, in which case happy to just leave as is :-)